cts/lab/CTSlab.py.in

```diff
@@ -37,6 +37,15 @@ def sig_handler(signum, frame) :
         sys.exit(1)
 
 
+def plural_s(n, uppercase=False):
+    if n == 1:
+        return ""
+    elif uppercase:
+        return "S"
+    else:
+        return "s"
+
+
```

Comment:
definitely more efficient like this.
if we pass the full string we could determine the case
and concatenate and spare a parameter.
hard to tell which one looks nicer in the code.
unfortunately something like returning an empty string
in case of zero will hardly be useful anywhere.